driver/bpf/fillers.h

```diff
@@ -151,21 +155,16 @@ FILLER_RAW(terminate_filler)
 					++state->n_drops_buffer_dir_file_enter;
 				}
 				break;
-			case PPME_SYSCALL_CLONE_11_E:
```

Comment:
Noticed that since we only include the current execve* events we probably also only need the latest clone/fork/* events, we can check more and clean up more when we touch stats in a cleanup PR again.